archive.php

```diff
@@ -0,0 +1,60 @@
+<?php
+require_once 'src/ViewHelpers/CarsViewHelper.php';
+require_once 'src/Models/CarsModel.php';
+require_once 'src/Entities/Car.php';
+require_once 'src/Models/MakeModel.php';
+require_once 'src/ViewHelpers/MakeViewHelper.php';
+
+$db = new PDO('mysql:host=db; dbname=Cars', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$cars_model = new CarsModel($db);
+$all_cars = $cars_model->getAllCars();
+
+$make_model = new MakeModel($db);
+$makes = $make_model->getAllMakes();
+
+?>
+
+<?php
+require_once 'src/ViewHelpers/CarsViewHelper.php';
+require_once 'src/Models/CarsModel.php';
+
+$db = new PDO('mysql:host=db; dbname=Cars', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$cars_model = new CarsModel($db);
```

Comment:
All of this can be binned. You've already got the requires in above, already have a PDO and already have a car model